application/src/main/java/com/softserve/lv460/application/service/DeviceTemplateService.java

```diff
@@ -40,7 +48,6 @@ public DeviceTemplate findById(Long id) {
   }
 
   public DeviceTemplate save(DeviceTemplate deviceTemplate) {
-    System.out.println(deviceTemplate);
```

Comment:
sout?

Reply:
Notice that this statement has already removed.